docs/linux/reporting_kernel_bugs.md

```diff
@@ -52,61 +52,121 @@ If you can't figure out the right fix, but have some understanding of the bug, p
 
-1. Report the bug publicly to kernel developers as described above and wait until a fix is committed. Alternatively, you can develop and send a fix yourself.
-2. Request a CVE from MITRE through [the web form](https://cveform.mitre.org/). Describe the bug details and add a link to the fix (from `patchwork.kernel.org`, `git.kernel.org` or `github.com`) in the request.
-3. Once a CVE is assigned, send the bug details, the CVE number and a link to the fix to `oss-security@lists.openwall.com`.
+4. `cve@kernel.org` - https://www.kernel.org/doc/html/latest/process/cve.html
 
-### Reporting major security bugs
+    Official kernel.org CNA. CVE assignment team has own procedure of manual voting whether the bug is CVE eligible.
 
-To report major security bugs (such as LPE, remote DOS, remote info leak or RCE):
+### Vulnerability definition
 
-1. Understand the bug and develop a patch with a fix if possible. Optionally develop a proof-of-concept exploit.
-2. Notify `security@kernel.org`:
-    * Describe vulnerability details, include the proposed patch and optionally the exploit.
-    * Ask for 7 days of embargo.
-    * Work on the patch together with the `security@kernel.org` members.
-3. Notify `linux-distros@vs.openwall.org`:
-    * Describe vulnerability details, include the proposed patch and optionally the exploit.
-    * Ask them to assign a CVE number.
-    * Ask for 7 days of embargo.
-4. Wait 7 days for linux distros to apply the patch.
-5. Ask `linux-distros@vs.openwall.org` to make the CVE description public and roll out the updated kernels.
-6. Send the fix upstream:
-    * Mention the CVE number in the commit message.
-    * Mention syzkaller in the commit message.
-7. Notify `oss-security@lists.openwall.com`:
-    * Describe vulnerability details, include a link to the committed patch.
-8. Wait 1-3 days for people to update their kernels.
-9. Optionally publish the exploit on `oss-security@lists.openwall.com`.
+From [cve.org](https://www.cve.org/resourcessupport/allresources/cnarules):
+
+> A vulnerability is an instance of one or more weaknesses in a Product that can be exploited,
+> causing a negative impact to confidentiality, integrity, or availability; a set of conditions
+> or behaviors that allows the violation of an explicit or implicit security policy.
+
+Translating to the kernel vulnerability definition per [Greg Kroah-Hartman](https://youtu.be/KumwRn1BA6s?t=203):
+> * Any user-triggerable crash or reboot
+> * Memory UAF / leak / overflow (even 1 byte)
+> * Incorrect boundary checks
+> * Denial of service
+> * Logic errors & lots of other things*
+
+*Data corruption/loss, performance issues, bugs that are not externally triggered are NOT considered as the kernel vulnerability.
+
+### Linux kernel security bug reporting
+
+Instructions are composed according to the [kernel.org instructions](https://docs.kernel.org/process/security-bugs.html),
```

Comment:
"These instructions are based on the kernel.org guidelines, which do not require CVE assignment. CVE assignment instructions are provided below."

If you choose to pick this change, remove the paragraph preceding `### CVE assignment` below.

Reply:
Done